Google.Api.Generator/Generation/ServiceSettingsCodeGenerator.cs

```diff
@@ -35,9 +36,21 @@ internal class ServiceSettingsCodeGenerator
 
         private ClassDeclarationSyntax Generate()
         {
-            var cls = Class(Public | Sealed | Partial, _svc.SettingsTyp, baseType: _ctx.Type<ServiceSettingsBase>());
-            // TODO: Fill the settings class.
+            var cls = Class(Public | Sealed | Partial, _svc.SettingsTyp, baseType: _ctx.Type<ServiceSettingsBase>())
+                .WithXmlDoc(XmlDoc.Summary("Settings for a ", _ctx.Type(_svc.ClientAbstractTyp), "."));
```

Comment:
Does the Java generator handle vowels properly, e.g. "an ImageAnnotatorClient" instead of "a ImageAnnotatorClient"? Maybe add a TODO?